epochStart/metachain/validators_test.go

```diff
@@ -118,8 +120,10 @@ func verifyMiniBlocks(bl *block.MiniBlock, infos []*state.ValidatorInfo, marshal
 		vi := validatorCopy[i]
 		unmarshaledVi := &state.ValidatorInfo{}
 		_ = marshalizer.Unmarshal(unmarshaledVi, txHash)
-
 		if !reflect.DeepEqual(unmarshaledVi, vi) {
+			log.Warn("verifyMiniBlocks existed on second if",
```

Comment:
change this log message to something that suggests wrong type

Reply:
removed. was just for testing purposes